### ISSUE TITLE
关于assert len(d_hiddens)==len(d_outs)==len(res_project)

### ISSUE THREAD

**Issue by baosir529a** [kiva12138/CubeMLP#7]
为什么运行的时候assert len(d_hiddens)==len(d_outs)==len(res_project)不等长，我得将这一句删除后才能运行

**Comment by kiva12138** [comment]
如果采用Readme中的参数运行的话应该不会出错，这三个变量（列表类型）的长度取决于运行时的参数`--d_hiddens 50-3-128=10-3-32  --d_outs 50-3-128=10-3-32 --res_project 1-1`，其中前两个参数的长度用=隔开，而最后一个参数的长度由-隔开。

**Comment by baosir529a** [comment]
谢谢学长，成功运行了，我一直使用的是默认的运行，学艺不精：（

**Comment by baosir529a** [comment]
感谢学长，成功运行了，祝您学业顺利！生活快乐！


喜羊羊
***@***.***



&nbsp;






------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "kiva12138/CubeMLP"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2023年8月26日(星期六) 下午4:28
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [kiva12138/CubeMLP] 关于assert len(d_hiddens)==len(d_outs)==len(res_project) (Issue #7)





 
如果采用Readme中的参数运行的话应该不会出错，这三个变量（列表类型）的长度取决于运行时的参数--d_hiddens 50-3-128=10-3-32  --d_outs 50-3-128=10-3-32 --res_project 1-1，其中前两个参数的长度用=隔开，而最后一个参数的长度由-隔开。

**Comment by baosir529a** [comment]
喜羊羊
***@***.***



&nbsp;我将mosi改为mosei后运行(python Train.py --dataset mosei_SDK --batch_size 2 --features_compose_t mean --features_compose_k cat --d_hiddens 50-3-128=10-3-32&nbsp; --d_outs 50-3-128=10-3-32 --res_project 1-1 --bias --ln_first&nbsp; --dropout_mlp 0.1-0.1-0.1 --dropout 0.1-0.1-0.1-0.1 --bert_freeze part --bert_lr_rate 0.01&nbsp; --learning_rate 4e-3)会出现(assert video in mosei_v_features,AssertionError）的错误，请问如何运行mosei数据集







------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "kiva12138/CubeMLP"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2023年8月26日(星期六) 下午5:46
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [kiva12138/CubeMLP] 关于assert len(d_hiddens)==len(d_outs)==len(res_project) (Issue #7)





 
Closed #7 as completed.

**Comment by kiva12138** [comment]
每个数据集的视频特征和视频特征都不一样，可以参照`DataLoaderCMUSDK.py`文件里的特征调整你的`--video --audio`参数
